src/www/app/outline_server_repository/server.ts

```diff
@@ -69,6 +73,8 @@ export class OutlineServer implements Server {
   }
 
   get address() {
+    if (!this.sessionConfig) return '';
```

Comment:
I think we could show the config location here for dynamic servers.

Reply:
No, i'm saying this is the serverAddress, not the onlineConfigAdress